addons/point_of_sale/views/pos_payment_views.xml

```diff
@@ -8,14 +8,14 @@
                 <sheet>
                     <group>
                         <field name="currency_id" invisible="1" />
-                        <field name="name" />
-                        <field name="amount" />
-                        <field name="pos_order_id" readonly="1"/>
-                        <field name="payment_method_id" readonly="1"/>
-                        <field name="card_type" readonly="1" invisible="not card_type"/>
-                        <field name="cardholder_name" readonly="1" invisible="not cardholder_name"/>
-                        <field name="transaction_id" readonly="1" invisible="not transaction_id"/>
-                        <field name="session_id" />
+                        <field name="name" invisible="1"/>
```

Comment:
Do we need this line ? And do we need the line n°10 ?

Reply:
yes we can remove both lines since it is always invisible